### ISSUE TITLE
Some Links not rendering properly

### ISSUE THREAD

**Issue by thepaperpilot** [mikeage/prpltwtr#22]
The star that links to the tweet, as well as any @ mention links, don't link, and instead appear with something like "(prpltwtr:///user?account=thepaperpilot...&protocol_id=prpl-twtr)"
The links work properly when viewing the logs.

example:
(03:36:03 PM) HobbyGameDev: RT @akaJohnMore (prpltwtr:///user?account=aThePaperPilot&text=%40akaJohnMore&protocol_id=prpl-twitter): @HobbyGameDev (prpltwtr:///user?account=aThePaperPilot&text=%40HobbyGameDev&protocol_id=prpl-twitter) left is Clash from 1 and a half year ago, right my current (WIP) project http://t.co/ARXAYDStoD \* (prpltwtr:///actions?account=aThePaperPilot&user=HobbyGameDev&id=577931424948350976&text=RT%20%40akaJohnMore%3A%20%40HobbyGameDev%20left%20is%20Clash%20from%201%20and%20a%20half%20year%20ago%2C%20right%20my%20current%20%28WIP%29%20project%20http%3A%2F%2Ft.co%2FARXAYDStoD&conv_type=2&conv_name=Timeline%3A%20Home&protocol_id=prpl-twitter)

**Comment by mikeage** [comment]
Do you have the "GtkPrplTwtr" plugin activated?

**Comment by thepaperpilot** [comment]
Yes. Icons are enabled, size 32

**Comment by thepaperpilot** [comment]
Also I should mention, I compiled from source (and am up to date with all current commits), and am on linux.

**Comment by thepaperpilot** [comment]
It also affects links to hashtags:
(02:02:01 PM) google: First upset of the tournament! http://t.co/N8nQu8ZFkn #MarchMadness (prpltwtr:///search?account=aThePaperPilot&text=%23MarchMadness&protocol_id=prpl-twitter) http://t.co/2P8DOlhRCI \* (prpltwtr:///actions?account=aThePaperPilot&user=google&id=578632534730383360&text=First%20upset%20of%20the%20tournament%21%20http%3A%2F%2Ft.co%2FN8nQu8ZFkn%20%23MarchMadness%20http%3A%2F%2Ft.co%2F2P8DOlhRCI&conv_type=2&conv_name=Timeline%3A%20Home&protocol_id=prpl-twitter)

Basically, any link that isn't a t.co

**Comment by thepaperpilot** [comment]
Here's an image, if that helps clarify how it appears on my screen.
Note: the :/ smilies are not the issue, switching to an icon theme without them doesn't fix the issue
![screenshot - 03192015 - 02 12 17 pm](https://cloud.githubusercontent.com/assets/3683148/6738750/095d76f4-ce42-11e4-8d80-4916c93325fa.png)
![screenshot - 03192015 - 02 12 38 pm](https://cloud.githubusercontent.com/assets/3683148/6738749/095d4116-ce42-11e4-9285-5520d10c15f2.png)

**Comment by mikeage** [comment]
interesting. I've seen this before; sometimes it's because the gtkprpltwtr plugin isn't enabled, but that's not the issue here. I've also heard about conflicts with other plugins that modify text, but I haven't been able to get a conclusive list of which plugins actually conflict. Can you think of anything that might be related?

**Comment by thepaperpilot** [comment]
I have a lot of plugins, unfortunately I tend to add them in groups, and can't pinpoint which one could be affecting this. Since I didn't have this issue until recently, I'll try disabling and waiting for more tweets to come in, and will report back if I find something.

edit: using "pidgin -nd | grep plugins" to help with plugins that don't appear in the plugins menu
edit2: My history plugin was misconfigured, and I fixed it while working on this. The links appear properly in the history as well
edit3: disabling everything except for gtkprpltwtr fixed it, so I'm going to work bottom->up

**Comment by thepaperpilot** [comment]
Found the culprit: Conversation Colors. 
hmmm, I use this to fix an issue with an irc channel I read that decided to format their messages black. I'll try and make it work, but a solution on the plugin side would be perfect.

**Comment by mikeage** [comment]
I'll try and take a look and see if I can find a way to let them work together.
